### PR TITLE
Compressed Photos of Memories Section

### PR DESCRIPTION
**Pull request by anuragsai97**
I've compressed the photos of memories section for improving load time.

### CODE REVIEW

**Comment by FlameFractal** [comment]
This is using ImageOptim ? I think this was discussed and it was finalized that Save to Web was going to be used, no?

**Comment by rhnvrm** [comment]
This was able to reduce ~2.5 MB. https://gtmetrix.com/reports/anuragsai97.github.io/45Cnkd42

Could you test if how much save to web is able to reduce the load time? I believe it will be similar. Plus, this image optim increases the number of requests we make.

Hey, please do the img.gs for performers images as well in this PR itself. Let's see the impact of this, anyways, LGTM 👍

**Comment by anuragsai97** [comment]
This morning ImageOptim was showing a 502:Bad Gateway error, hence the images of memories section could not be loaded. I fear if this happens in future it can affect the website. So, shall I download the images, compress and use them? For downloading the images shall I use the link meant to embed them or can anyone ask kondapivskrishnaprasanth to enable the download option in his breeze album.

File: index.html [hunk 1]
```diff
@@ -338,65 +338,65 @@ <h2 class=" text-center ">
                     <div class="col-md-3 col-xs-6 pad">
                        
                         
-                            <a href="https://c5.staticflickr.com/2/1474/24949578516_88b96ee0c3_k.jpg" data-lightbox="roadtrip" data-title="1" class="over">
-                                <img src="https://c5.staticflickr.com/2/1474/24949578516_88b96ee0c3_k.jpg" class="img-rounded width-100">
```
Comment (rhnvrm) [inline index.html:342]: @anuragsai97 use this link to download photos

**Comment by anuragsai97** [comment]
Ok

**Comment by anuragsai97** [comment]
I have made the necessary changes to the memories section.

**Comment by rhnvrm** [comment]
LGTM :+1:

**Comment by rhnvrm** [comment]
@anuragsai97 can you do the same thing for all other images as well if possible (sponsors and artists)

**Comment by anuragsai97** [comment]
Sure

**Pull request merged**